autoload/neomake/makers/ft/swift.vim

```diff
@@ -0,0 +1,11 @@
+function! neomake#makers#ft#swift#EnabledMakers()
+    return ['swiftlint']
+endfunction
+
+function! neomake#makers#ft#swift#swiftlint()
+    return {
+        \ 'args': ['lint', '--config', './.swiftlint.yml', '--quiet'],
```

Comment:
Yeah... I thought I had trouble with it if I didn't specify, but the linter _should_ find its own config. I'll give it another go and see.